frameworks/javascript/cw-2.js

```diff
@@ -356,6 +356,12 @@ try {
         Test.expect(false, msg, options);
       }
     },
+    assertFuzzyEquals: function (actual, expected, msg = "") {
```

Comment:
Or one could change `assertSimilar` to use this kind of check if both types are `number`s and not `int32`.